src/Error/BaseErrorHandler.php

```diff
@@ -15,7 +15,6 @@
 namespace Cake\Error;
 
 use Cake\Core\Configure;
-use Cake\Error\PHP7ErrorException;
```

Comment:
`PHP7ErrorException` is used on line 161, I had commented the same on a similar earlier PR too. Whatever IDE / tool you guys are using isn't smart enough.